### ISSUE TITLE
Release title different between interactive search and activity queue 

### ISSUE THREAD

**Issue by NicolasFerec** [Sonarr/Sonarr#4730]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I see a difference between the release title from interactive search and the one showing in the activity queue while downloading.

I use Prowlarr as source for my indexer (Ygg) and set it to rename the "multi" in "MULTI.FRENCH" in release title. 
When I do a interactive search on a episode, I get for example **The.Walking.Dead.S11E04.MULTI.FRENCH.1080p.WEB-DL.H264-AVON**, it is correctly detected as French language. 

![Capture1](https://user-images.githubusercontent.com/6369799/139778981-9c0dc1c3-fa20-4ed2-bf90-7b151e0b168e.PNG)

Then I click to add it to the download queue and go to the download queue and the release title is became **the.walking.dead.s11e04.multi.1080p.web.h264-avon**, probably the original name of the release before the indexer renamed it. And of corse, is detected as English.

![Capture2](https://user-images.githubusercontent.com/6369799/139778993-e8411091-0ec5-4613-a7c2-ec9da0f679c7.PNG)

### Expected Behavior

I expected that the release name (and the language detection based on it) was the same between the interactive search and the activity queue.

### Environment

```markdown
- OS: I don't know
- Sonarr:3.0.6.1342
- Docker Install: Yes, 3.0.6.1342-ls126 by linuxserver.io
- Using Reverse Proxy: Yes, SWAG by linuxserver
- Browser: Chrome
```

### What branch are you running?

Main

### Anything else?

2021-11-02 03:57:55.9|Debug|QBittorrent|Downloading torrent for episode 'The.Walking.Dead.S11E04.MULTI.FRENCH.1080p.WEB-DL.H264-AVON' finished (24097 bytes from https://prowlarr.ferecium.fr/2/download?apikey=(removed)&link=eG5GclJSckpJVkdlUEpKTlE5bHRKWjErZG5PYnBkZmJrd1FZY2d3MUg4dThmSDhLcmhheENEZ2MzV0JoWVRyemdpdGVUM0hYMFlOM0xSL1FyZURGN2xXS1VrMllVaTJwSnhhRHdIM2tNMXM9&file=The.Walking.Dead.S11E04.MULTI.FRENCH.1080p.WEB-DL.H264-AVON)
[...]
2021-11-02 03:58:01.0|Trace|HttpClient|Res: [GET] https://qbittorrent.ferecium.fr:443/api/v2/torrents/info?category=S%C3%A9rie: 200.OK (14 ms)
2021-11-02 03:58:01.0|Trace|HttpClient|Response content (9868 bytes): [{"added_on":1635357109,"amount_left":0,"auto_tmm":true,"availability":-1,"category":"SÃ©rie","completed":1709641886,"completion_on":1635357162,"content_path":"/shares/download/Complete/Koh-Lanta - 2021.La lÃ©gende.Ep09.HDLIGHT.1080P.X264.mkv","dl_limit":-1,"dlspeed":0,"downloaded":1713523541,"downloaded_session":0,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"4da021a5c275b488e1cd412a634cb8cf6d731ce6","last_activity":1635818061,"magnet_uri":"magnet:?xt=urn:btih:4da021a5c275b488e1cd412a634cb8cf6d731ce6&dn=Koh-Lanta%20-%202021.La%20l%c3%a9gende.Ep09.HDLIGHT.1080P.X264.mkv&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"Koh-Lanta - 2021.La lÃ©gende.Ep09.HDLIGHT.1080P.X264.mkv","num_complete":387,"num_incomplete":99,"num_leechs":0,"num_seeds":0,"priority":0,"progress":1,"ratio":6.330250223857298,"ratio_limit":-2,"save_path":"/shares/download/Complete/","seeding_time":460853,"seeding_time_limit":-2,"seen_complete":1635818124,"seq_dl":false,"size":1709641886,"state":"stalledUP","super_seeding":false,"tags":"","time_active":460905,"total_size":1709641886,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":10847032779,"uploaded_session":301259593,"upspeed":0},{"added_on":1635818764,"amount_left":0,"auto_tmm":true,"availability":-1,"category":"SÃ©rie","completed":2468012094,"completion_on":1635818839,"content_path":"/shares/download/Complete/The.Walking.Dead.S11E01.MULTi.1080p.DSNP.WEB-DL.DDP5.1.H264-FRATERNiTY","dl_limit":-1,"dlspeed":0,"downloaded":2474763310,"downloaded_session":2474763310,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"281559a54f25cfeb1cec40c3113974b819c69852","last_activity":1635818839,"magnet_uri":"magnet:?xt=urn:btih:281559a54f25cfeb1cec40c3113974b819c69852&dn=The.Walking.Dead.S11E01.MULTi.1080p.DSNP.WEB-DL.DDP5.1.H264-FRATERNiTY&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"The.Walking.Dead.S11E01.MULTi.1080p.DSNP.WEB-DL.DDP5.1.H264-FRATERNiTY","num_complete":322,"num_incomplete":14,"num_leechs":0,"num_seeds":0,"priority":0,"progress":1,"ratio":0,"ratio_limit":-2,"save_path":"/shares/download/Complete","seeding_time":2464,"seeding_time_limit":-2,"seen_complete":1635818997,"seq_dl":false,"size":2468012094,"state":"stalledUP","super_seeding":false,"tags":"","time_active":2539,"total_size":2468012094,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":0,"uploaded_session":0,"upspeed":0},{"added_on":1635778743,"amount_left":0,"auto_tmm":true,"availability":-1,"category":"SÃ©rie","completed":953619753,"completion_on":1635778810,"content_path":"/shares/download/Complete/Greys Anatomy - S18E04 - With a Little Help From My Friends - [VOSTFR][WEB-Rip][1080p][H265_10b][HE-AAC_5.1] - K-T.mkv","dl_limit":-1,"dlspeed":0,"downloaded":955435524,"downloaded_session":955435524,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"eb74ea2f197db492d31b9706be503dd38c4f7a70","last_activity":1635810605,"magnet_uri":"magnet:?xt=urn:btih:eb74ea2f197db492d31b9706be503dd38c4f7a70&dn=Greys%20Anatomy%20-%20S18E04%20-%20With%20a%20Little%20Help%20From%20My%20Friends%20-%20%5bVOSTFR%5d%5bWEB-Rip%5d%5b1080p%5d%5bH265_10b%5d%5bHE-AAC_5.1%5d%20-%20K-T.mkv&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"Greys Anatomy - S18E04 - With a Little Help From My Friends - [VOSTFR][WEB-Rip][1080p][H265_10b][HE-AAC_5.1] - K-T.mkv","num_complete":39,"num_incomplete":48,"num_leechs":0,"num_seeds":0,"priority":0,"progress":1,"ratio":4.491216799261485,"ratio_limit":-2,"save_path":"/shares/download/Complete","seeding_time":42493,"seeding_time_limit":-2,"seen_complete":1635810669,"seq_dl":false,"size":953619753,"state":"stalledUP","super_seeding":false,"tags":"","time_active":42559,"total_size":953619753,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":4291068076,"uploaded_session":4291068076,"upspeed":0},{"added_on":1635450134,"amount_left":0,"auto_tmm":true,"availability":-1,"category":"SÃ©rie","completed":488651247,"completion_on":1635450168,"content_path":"/shares/download/Complete/Grey's.Anatomy.S18E04.With.a.Little.Help.From.My.Friends.1080p.WEBRip.6CH.x265.HEVC.mkv","dl_limit":-1,"dlspeed":0,"downloaded":492004273,"downloaded_session":0,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"1171a53ac4855fe5c4517988ef9a503037bb2be7","last_activity":1635803260,"magnet_uri":"magnet:?xt=urn:btih:1171a53ac4855fe5c4517988ef9a503037bb2be7&dn=Grey%27s.Anatomy.S18E04.With.a.Little.Help.From.My.Friends.1080p.WEBRip.6CH.x265.HEVC.mkv&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"Grey's.Anatomy.S18E04.With.a.Little.Help.From.My.Friends.1080p.WEBRip.6CH.x265.HEVC.mkv","num_complete":241,"num_incomplete":96,"num_leechs":0,"num_seeds":0,"priority":0,"progress":1,"ratio":14.472508024335797,"ratio_limit":-2,"save_path":"/shares/download/Complete/","seeding_time":367847,"seeding_time_limit":-2,"seen_complete":1635796604,"seq_dl":false,"size":488651247,"state":"stalledUP","super_seeding":false,"tags":"","time_active":367880,"total_size":488651247,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":7120535789,"uploaded_session":518097555,"upspeed":0},{"added_on":1635821875,"amount_left":2464330169,"auto_tmm":true,"availability":10.006999969482422,"category":"SÃ©rie","completed":30736384,"completion_on":-3600,"content_path":"/shares/download/Complete/the.walking.dead.s11e04.multi.1080p.web.h264-avon.mkv","dl_limit":-1,"dlspeed":4186070,"downloaded":24830920,"downloaded_session":30781842,"eta":1868,"f_l_piece_prio":false,"force_start":false,"hash":"6917e4fb6e4a8b14daaa06fa091cac0be4c3fcde","last_activity":1635821879,"magnet_uri":"magnet:?xt=urn:btih:6917e4fb6e4a8b14daaa06fa091cac0be4c3fcde&dn=the.walking.dead.s11e04.multi.1080p.web.h264-avon.mkv&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"the.walking.dead.s11e04.multi.1080p.web.h264-avon.mkv","num_complete":272,"num_incomplete":2,"num_leechs":1,"num_seeds":10,"priority":1,"progress":0.01231886338384177,"ratio":0,"ratio_limit":-2,"save_path":"/shares/download/Complete/","seeding_time":0,"seeding_time_limit":-2,"seen_complete":1635821879,"seq_dl":false,"size":2495066553,"state":"downloading","super_seeding":false,"tags":"","time_active":3,"total_size":2495066553,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":0,"uploaded_session":0,"upspeed":0},{"added_on":1635789726,"amount_left":0,"auto_tmm":true,"availability":-1,"category":"SÃ©rie","completed":8454671096,"completion_on":1635812866,"content_path":"/shares/download/Complete/Manifest.SAISON.02.FRENCH.1080P.TVripHD1080.MKV.h265.AAC-2-OLRIK","dl_limit":-1,"dlspeed":0,"downloaded":8457144520,"downloaded_session":8457144520,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"12496f4769cfb27b6ccf74c3e3f38e880ac692a7","last_activity":1635821879,"magnet_uri":"magnet:?xt=urn:btih:12496f4769cfb27b6ccf74c3e3f38e880ac692a7&dn=Manifest.SAISON.02.FRENCH.1080P.TVripHD1080.MKV.h265.AAC-2-OLRIK&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"Manifest.SAISON.02.FRENCH.1080P.TVripHD1080.MKV.h265.AAC-2-OLRIK","num_complete":17,"num_incomplete":2,"num_leechs":1,"num_seeds":0,"priority":0,"progress":1,"ratio":4.637104872957758,"ratio_limit":-2,"save_path":"/shares/download/Complete","seeding_time":9013,"seeding_time_limit":-2,"seen_complete":1635820378,"seq_dl":false,"size":8454671096,"state":"uploading","super_seeding":false,"tags":"","time_active":32151,"total_size":8454671096,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":39216666065,"uploaded_session":39216712062,"upspeed":60871},{"added_on":1635789728,"amount_left":0,"auto_tmm":true,"availability":-1,"category":"SÃ©rie","completed":12739294081,"completion_on":1635816898,"content_path":"/shares/download/Complete/Manifest.SAISON.01.FRENCH.1080P.TVripHD1080.MKV.h265.AC-3-5.1-OLRIK","dl_limit":-1,"dlspeed":0,"downloaded":12750584195,"downloaded_session":12750584195,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"23fc93aaa1469445f67761a9e77517c8f3a25ff3","last_activity":1635821879,"magnet_uri":"magnet:?xt=urn:btih:23fc93aaa1469445f67761a9e77517c8f3a25ff3&dn=Manifest.SAISON.01.FRENCH.1080P.TVripHD1080.MKV.h265.AC-3-5.1-OLRIK&tr=http%3a%2f%2ftracker.p2pconnect.net%3a8080%2fJCitMflEtLAdeF6p7EqVY0TDAZdX0cKz%2fannounce","max_ratio":-1,"max_seeding_time":-1,"name":"Manifest.SAISON.01.FRENCH.1080P.TVripHD1080.MKV.h265.AC-3-5.1-OLRIK","num_complete":20,"num_incomplete":5,"num_leechs":3,"num_seeds":0,"priority":0,"progress":1,"ratio":1.4262639135492567,"ratio_limit":-2,"save_path":"/shares/download/Complete","seeding_time":4981,"seeding_time_limit":-2,"seen_complete":1635821458,"seq_dl":false,"size":12739294081,"state":"uploading","super_seeding":false,"tags":"","time_active":32150,"total_size":12739294081,"tracker":"http://tracker.p2pconnect.net:8080/JCitMflEtLAdeF6p7EqVY0TDAZdX0cKz/announce","trackers_count":1,"up_limit":-1,"uploaded":18185698114,"uploaded_session":18185928481,"upspeed":661712}]
2021-11-02 03:58:01.0|Debug|Parser|Parsing string 'the.walking.dead.s11e04.multi.1080p.web.h264-avon.mkv'
2021-11-02 03:58:01.0|Trace|Parser|^(?<title>.+?)(?:(?:[-_\W](?<![()\[!]))+S?(?<season>(?<!\d+)(?:\d{1,2})(?!\d+))(?:[ex]|\W[ex]){1,2}(?<episode>\d{2,3}(?!\d+))(?:(?:\-|[ex]|\W[ex]|_){1,2}(?<episode>\d{2,3}(?!\d+)))*)\W?(?!\\)
2021-11-02 03:58:01.0|Debug|Parser|Episode Parsed. the walking dead - S11E04 
2021-11-02 03:58:01.0|Debug|Parser|Language parsed: English
2021-11-02 03:58:01.0|Debug|QualityParser|Trying to parse quality for the.walking.dead.s11e04.multi.1080p.web.h264-avon.mkv
2021-11-02 03:58:01.0|Debug|Parser|Quality parsed: WEBDL-1080p v1
2021-11-02 03:58:01.0|Debug|Parser|Release Group parsed: avon

**Comment by NicolasFerec** [comment]
To sum up, my issue is that the mechanism who decide whether it should import an episode after download or not seems to be based on the downloaded file name rather than on the "Release title" provided by the Indexer.

**Comment by bakerboy448** [comment]
> To sum up, my issue is that the mechanism who decide whether it should import an episode after download or not seems to be based on the downloaded file name rather than on the "Release title" provided by the Indexer.

Correct this is intentional / expected and they are two distinct black boxes.  Specifically to avoid importing files where where the file is not what the what the release claimed to be
